DEVELOPMENT.md

```diff
@@ -808,7 +718,37 @@ We use [Cypress](https://cypress.io) for running frontend-based end-to-end tests
 
 #### Running frontend end-to-end tests locally
 
-Make sure to [build the frontend assets](#building-the-frontend) and run `tools/bin/mage dev:initStack dev:sqlDump` to create a seed database which the tests can reset the database to in between runs. To run the stack when working on end-to-end tests, use the `tools/bin/mage dev:startDevStack` command. This will run the runs The Things Stack with proper configuration for the end-to-end tests. Note: this command does not output anything, but logs are written to `.cache/devStack.log`.
+The preferred way to run the end-to-end tests is to use the interactive launcher tool with:
+
+```bash
+$  tools/bin/mage dev:serveDevWebui
+```
```

Comment:
This might be a bit confusing, at least it was for me when reading it I ended up attempting to run the `serveDevWebui` before the step 1.

This could be replaced with a single phrase that references to the `#interactive-development-stack-launcher-tool` section of `development.md`